### PR TITLE
APEXMALHAR-2381 Change WindowManager for performance issues in Kinesi…

### PR DESCRIPTION
**Pull request by deepak-narkhede**
For performance benefits change existing windowDataManager. If FSWindowManager is required we can very well set through property. 

Tested with Kinesis to S3 app.

### CODE REVIEW

File: contrib/src/main/java/com/datatorrent/contrib/kinesis/AbstractKinesisInputOperator.java [hunk 1]
```diff
@@ -147,7 +146,7 @@ public KinesisPair(F first, S second)
 
   public AbstractKinesisInputOperator()
   {
-    windowDataManager = new FSWindowDataManager();
+    windowDataManager = new WindowDataManager.NoopWindowDataManager();
```
Comment (chaithu14) [inline contrib/src/main/java/com/datatorrent/contrib/kinesis/AbstractKinesisInputOperator.java]: Could you please add some mock test for this change?

Comment (deepak-narkhede) [reply contrib/src/main/java/com/datatorrent/contrib/kinesis/AbstractKinesisInputOperator.java]: @chaithu14 . Added test.

File: contrib/src/test/java/com/datatorrent/contrib/kinesis/KinesisInputOperatorTest.java [hunk 2]
```diff
@@ -91,6 +95,30 @@ public void setConnected(boolean flag)
...
+      {
+      }
+
+      @Override
+      public void teardown()
+      {
+      }
+    });
+    testMeta.operator = inputOperator;
+    Assert.assertFalse("Default behaviour of WindowDataManager changed",
```
Comment (chaithu14) [inline contrib/src/test/java/com/datatorrent/contrib/kinesis/KinesisInputOperatorTest.java]: Specify this condition under assertTrue.

Comment (deepak-narkhede) [reply contrib/src/test/java/com/datatorrent/contrib/kinesis/KinesisInputOperatorTest.java]: @chaithu14 .Done.

**Comment by chaithu14** [comment]
Looks good to me.

**Pull request merged**